client/components/happychat/notices.jsx

```diff
@@ -16,22 +16,29 @@ import {
 	HAPPYCHAT_CHAT_STATUS_ABANDONED,
 	getHappychatStatus,
 	getHappychatConnectionStatus,
+	isHappychatServerReachable,
 } from 'state/happychat/selectors';
 
 /*
  * Renders any notices about the chat session to the user
  */
 class Notices extends Component {
 	statusNotice() {
-		const { connectionStatus, chatStatus, translate } = this.props;
+		const { isServerReachable, connectionStatus, chatStatus, translate } = this.props;
+
+		if ( ! isServerReachable ) {
+			return translate( "We're having trouble connecting to chat. Please check your internet connection while we try to reconnect…" );
+		}
 
 		switch ( connectionStatus ) {
```

Comment:
We could use a shared constant to communicate [connectionStatus values](https://github.com/Automattic/wp-calypso/blob/eb428f37046b8e3dbaba7abe627287c176919eb1/client/state/happychat/reducer.js#L148) between this and the reducer.